pkgs/by-name/st/stalwart-mail/package.nix

```diff
@@ -141,10 +145,11 @@ rustPlatform.buildRustPackage (finalAttrs: {
     "--skip=smtp::inbound::antispam::antispam"
     # Failed to read system DNS config: io error: No such file or directory (os error 2)
     "--skip=smtp::inbound::vrfy::vrfy_expn"
+    # Failed to read system DNS config: io error: No such file or directory (os error 2)
+    "--skip=smtp::inbound::vrfy::vrfy_expn"
     # thread 'smtp::management::queue::manage_queue' panicked at tests/src/smtp/inbound/mod.rs:45:23:
     # No queue event received.
     # NOTE: Test unreliable on high load systems
-    "--skip=smtp::management::queue::manage_queue"
```

Comment:
The comment above should be removed too

Reply:
Whoops that shouldn't actually be removed. I played around with the tests and forgot to revert that. Fixed now.